config/config_test.go

```diff
@@ -1,4 +1,4 @@
-package main
+package configs
```

Comment:
```suggestion
package config
```

Reply:
Yup! No idea how the IDE didn't catch this. 😅

Reply:
This just goes to show, we need way more unit tests 🤣